ci-operator/step-registry/aws/provision/kms-key/aws-provision-kms-key-commands.sh

```diff
@@ -67,33 +67,45 @@ cat > ${key_policy} << EOF
 EOF
 
 
-ts=$(date +%m%d%H%M%S)
-alias_name="${CLUSTER_NAME}-${ts}"
-echo "Creating KMS key: $alias_name"
-echo "Policy:"
-cat $key_policy
+function create_kms_key()
+{
+    local id_output=$1
+    local arn_output=$2
 
-key_output=${ARTIFACT_DIR}/key_output.json
+    local key_arn key_id output ts
 
-aws --region $REGION kms create-key --description "Prow CI $alias_name" \
-  --output json \
-  --policy "$(cat $key_policy | jq -c)" > "${key_output}" || exit 1
+    ts=$(date +%m%d%H%M%S)
+    key_alias="alias/prowci-${CLUSTER_NAME}-${ts}-${RANDOM}"
+    output=$(mktemp)
 
-key_arn=$(cat "${key_output}" | jq -r '.KeyMetadata.Arn')
-key_id=$(cat "${key_output}" | jq -r '.KeyMetadata.KeyId')
+    echo "Creating KMS key: $key_alias"
+    aws --region $REGION kms create-key --description "Prow CI $key_alias" --output json \
+        --policy "$(cat $KEY_POLICY | jq -c)" > "${output}" || return 1
+    
+    key_arn=$(cat "${output}" | jq -r '.KeyMetadata.Arn')
+    key_id=$(cat "${output}" | jq -r '.KeyMetadata.KeyId')
 
-if [[ "${key_arn}" == "" ]] || [[ "${key_arn}" == "null" ]] || [[ "${key_id}" == "" ]] || [[ "${key_id}" == "null" ]]; then
-  echo "Failed to create KMS key."
-  exit 1
-fi
+    if [[ "${key_arn}" == "" ]] || [[ "${key_arn}" == "null" ]] || [[ "${key_id}" == "" ]] || [[ "${key_id}" == "null" ]]; then
+        echo "Failed to create KMS key."
+        return 1
+    fi
 
-echo "Created key $key_arn"
+    echo $key_arn > "${id_output}"
+    echo $key_id > "${arn_output}"
```

Comment:
should be ->
```suggestion
    echo $key_arn > "${arn_output}"
    echo $key_id > "${id_output}"
```
@yunjiang29 @jianlinliu ^^